WooCommerce/Classes/ViewRelated/Products/Add Product/FirstProductCreatedView.swift

```diff
@@ -51,10 +53,14 @@ struct FirstProductCreatedView: View {
                 Text(Localization.message)
                     .secondaryBodyStyle()
                     .multilineTextAlignment(.center)
-                Button(Localization.shareAction,
-                       action: onSharingProduct)
+
+                if showShareProductButton {
+                    Button(Localization.shareAction,
+                           action: onSharingProduct)
                     .buttonStyle(PrimaryButtonStyle())
                     .padding(.horizontal)
+                }
```

Comment:
Nit: We can also use the `.renderedIf` wrapper to get rid of the if statement:
```suggestion
                    Button(Localization.shareAction,
                           action: onSharingProduct)
                    .buttonStyle(PrimaryButtonStyle())
                    .padding(.horizontal)
                    .renderedIf(showShareProductButton)
```

Reply:
Done in 1b34f72ffbd67976ba3155a05fcc5f4b14ec2cab